core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -1308,7 +1312,9 @@ class GroupCoordinator(val brokerId: Int,
     completeAndScheduleNextHeartbeatExpiration(group, member)
 
     val knownStaticMember = group.get(newMemberId)
-    group.updateMember(knownStaticMember, protocols, responseCallback)
+    val oldRebalanceTimeoutMs = knownStaticMember.rebalanceTimeoutMs
+    val oldSessionTimeoutMs = knownStaticMember.sessionTimeoutMs
+    group.updateMember(knownStaticMember, protocols, rebalanceTimeoutMs, sessionTimeoutMs, responseCallback)
     val oldProtocols = knownStaticMember.supportedProtocols
```

Comment:
Not related to your PR, but it seems that we should get the old protocols before updating the member, no? We may have a gap in our testing here.

Reply:
Fixed